src/sentry/static/sentry/app/components/platformPicker.tsx

```diff
@@ -43,9 +48,14 @@ class PlatformPicker extends React.Component {
     const {category} = this.state;
     const currentCategory = categoryList.find(({id}) => id === category);
 
-    const subsetMatch = ({id}) => id.includes(this.state.filter.toLowerCase());
-    const categoryMatch = platform =>
-      category === 'all' || currentCategory.platforms.includes(platform.id);
+    const filter = this.state.filter.toLowerCase();
+
+    const subsetMatch = (platform: PlatformIntegration) =>
+      platform.id.includes(filter) || platform.name.toLowerCase().includes(filter);
+
+    const categoryMatch = (platform: PlatformIntegration) =>
+      category === 'all' ||
+      (currentCategory?.platforms as undefined | string[])?.includes(platform.id);
```

Comment:
Would it be more legible to do `{ id: 'all', name: t('All'), platforms: [] }` on Line 21?